model/mod_list.py

```diff
@@ -319,7 +319,7 @@ def eventFilter(self, source_object: QObject, event: QEvent) -> None:
                         # Open Steam URI with Steam action
                         elif (
                             action == open_mod_steam_action
-                        ):  # ACTION: Open steam:// uri in Steam
+                        ):  # ACTION: Open steam:// url in Steam
```

Comment:
This is not a typo...

https://en.wikipedia.org/wiki/Uniform_Resource_Identifier